Makefile

```diff
@@ -39,16 +39,3 @@ configure:
 .PHONY: build doc test all install uninstall reinstall clean distclean configure
 
 # OASIS_STOP
```

Comment:
Thanks for noticing this - I accidentally clobbered it when committing the results of `oasis setup` for the nine zillionth time.  I've restored it.